app/views/collections/_filters.html.erb

```diff
@@ -100,36 +74,26 @@
       <dd>
         <ul>
           <li>
-            <%= label_tag "collection_filters_challenge_type_gift_exchange" do %>
-              <%= radio_button_tag "collection_filters[challenge_type]",
-                  "gift_exchange",
-                  params[:collection_filters][:challenge_type] == "gift_exchange"
-              %>
+            <%= f.label "challenge_type_gift_exchange" do %>
+              <%= f.radio_button "challenge_type", "GiftExchange" %>
```

Comment:
Changing from `gift_exchange` to `GiftExchange` as the value appears to have broken this radio button. I think the label needs to change to `challenge_type_giftexchange`, and similarly with the label for the prompt meme radio button below.

Reply:
I'm a bit confused. It seems to be working for me. It's passing the value to the controller and filtering the results correctly. It's also highlighting the correct option on reload. Is there something I'm overlooking?

Reply:
In my browser, clicking on the label won't select the radio button. Even clicking on the radio button doesn't seem to work. I think it's because the real radio button is 1px, while the thing that looks like a radio button is actually a `<span>` styled to look like a radio button, and it can't control the radio button unless the label is correct.

(You need skins enabled to encounter this bug.)

Reply:
Ah, gotcha. I didn't have skins enabled. Thanks for the explanation. Will be changed soon. :)